packages/indy-sdk-to-askar-migration/tests/migrate.test.ts

```diff
@@ -0,0 +1,56 @@
+import { AskarModule } from '@aries-framework/askar'
+import { utils, KeyDerivationMethod, Agent } from '@aries-framework/core'
+import { IndySdkModule } from '@aries-framework/indy-sdk'
+import { agentDependencies } from '@aries-framework/node'
+import { ariesAskar } from '@hyperledger/aries-askar-nodejs'
+import { registerAriesAskar } from '@hyperledger/aries-askar-shared'
+import indy from 'indy-sdk'
+import { homedir } from 'os'
+
+import { IndySdkToAskarMigrationUpdater } from '../src'
+
+describe('Migrate', () => {
+  const config = {
+    label: 'test-agent',
+    walletConfig: {
+      id: `walletwallet.0-${utils.uuid()}`,
+      key: 'GfwU1DC7gEZNs3w41tjBiZYj7BNToDoFEqKY6wZXqs1A',
+      keyDerivationMethod: KeyDerivationMethod.Raw,
+    },
+  }
+
+  const oldAgent = new Agent({
+    config,
+    modules: { indySdk: new IndySdkModule({ indySdk: indy }) },
+    dependencies: agentDependencies,
+  })
+
+  const newAgent = new Agent({
+    config,
+    modules: { askar: new AskarModule() },
+    dependencies: agentDependencies,
+  })
+
+  const oldDbPath = `${homedir()}/.indy_client/wallet/${oldAgent.config.walletConfig?.id}/sqlite.db`
+
+  beforeAll(() => {
+    registerAriesAskar({ askar: ariesAskar })
+  })
+
+  test('indy-sdk sqlite to aries-askar sqlite', async () => {
```

Comment:
test feel light. Would like to have some of the AFJ specific transformations tests. Will you address that in a separate PR?

Reply:
Not needed before merging

Reply:
Yeah I was thinking about this and I was not super sure what the best way to get mock wallets was.

Reply:
I added one that checks the restoring of the wallet, but I 100% agree there need to be better tests.